src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -10,8 +10,9 @@ namespace Microsoft.Azure.WebJobs.ServiceBus
     /// Setup an 'output' binding to an EventHub. This can be any output type compatible with an IAsyncCollector.
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
+    [ConnectionProvider(typeof(ServiceBusAccountAttribute))]
```

Comment:
Remove this - ServiceBus doesn't really apply here, and I don't think we need to support method/class level overrides. The SBAA is only for service bus.